packages/block-library/src/navigation/edit.js

```diff
@@ -55,6 +57,7 @@ function Navigation( {
 	//
 	// HOOKS
 	//
+	/* eslint-disable @wordpress/no-unused-vars-before-return */
```

Comment:
which variable is unused? can we mark just that one or are all impacted?

Reply:
Look like a regression introduced by the revert. My bad. Removed and there are no warnings so we're all clear.